data/1js2002.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "Shiraan",
+  "type": "personal",
+  "bio": "MERN stack developer",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/1js2002",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://github.com/1js2002",
```

Did you mean with **GitHub** here!?

```suggestion
      "name": "Follow me on GitHub",
      "url": "https://github.com/1js2002",
```